data/faq.html

```diff
@@ -383,2403 +383,2403 @@ <h4 lang="uk">
+          the Car, Pedestrian, Subway, Bike or Taxi icon at the top of the
+          screen. To start following the route tap “Start” button. Press the
+          arrow sign in the lower right corner of the screen and tap Stop to
+          finish the route.
         </p>
 
         <p>
-          The more people contribute, the more detailed maps we get. We believe
-          that the most detailed map of the whole world, created by the open
-          community, is just a matter of time.
+          You can choose a different starting point (Route from
+          button) to preview a route, but navigation is only available from your
+          current location.
         </p>
 
-        <p><b>Notes:</b></p>
-
-        <ul>
-          <li>
-            <p>
-              You can also add new places, edit existing POI and building info
-              (addresses, opening hours, names) directly via Organic Maps. Once
-              you log in with OSM account, your edits will be automatically
-              uploaded to OSM. Please be careful when editing, as your edits
-              will be visible to all other users.
-            </p>
-          </li>
-          <li>
-            <p>
-              The OpenStreetMap database is updated every minute. We aim to
-              update maps in the app 1-4 times per month. If you modified
-              something in OSM, your edits will appear in future maps updates.
-            </p>
-          </li>
-        </ul>
-      </dd>
-
-      <dd lang="ru">
         <p>
-          В Organic Maps мы делаем само приложение и сложные алгоритмы. Наш
-          источник данных –
-          <a href="https://www.openstreetmap.org/">OpenStreetMap</a> (OSM), это
-          открытый проект, как Википедия, только для карт.
+          You can add up to 3 intermediate points to a route. To add an
+          intermediate point: create a route between start and destination →
+          then tap a point on the map and tap “Add stop”.
         </p>
 
         <p>
-          Если вы заметили неточности или увидели, что отсутствуют какие-либо
-          детали, вы можете
-          <a href="https://www.openstreetmap.org/note/new">сообщить об этом</a>
-          редакторам OSM или
-          <a href="https://www.openstreetmap.org/user/new"
-            >зарегистрироваться</a
-          >
-          на сайте OpenStreetMap и отредактировать карту.
+          You can change the settings of the car route and select road types
+          that you would like to avoid (tolls, unpaved roads, motorways,
+          ferries). Open the app settings → Routing options → Toggle On needed
+          options.
         </p>
+      </dd>
 
+      <dd lang="ru">
         <p>
-          Чем больше небезразличных людей, тем точнее будут карты для всех нас.
-          Мы в команде считаем, что подробные карты всей нашей планеты – это
-          всего лишь вопрос времени.
+          Убедитесь, что ваше местоположение определено и выберите пункт
```

Comment:
There are so many edits in this file, so it is impossible to review it.

I propose not to change it in this PR and create a separate one, where the first commit will have formatting fixes (if they are really necessary), and the second commit will have actual changes.

Reply:
removed html from PR